1-exercises/E-while-loop-with-array/exercise.js

```diff
@@ -18,6 +18,11 @@ const BIRTHDAYS = [
 
 function findFirstJulyBDay(birthdays) {
     // TODO
+    for (let i = 0 ; i < BIRTHDAYS.length; i++)
+    if( BIRTHDAYS[i] ===  "July 11th" ) {
```

Comment:
So I'm not sure if you have worked with string methods yet, but this method is a bit specific to that array. The question is 'find the first birthday in July', so I think we would like it to work for other birthdays. Something like 
`if(BIRTHDAYS[i].includes('July')` would work for an string containing 'July' :)

Reply:
Thank you * * I have not  worked with string methods yet.